sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-resource-server-by-filter/README.md

```diff
@@ -87,57 +87,53 @@ As a first step you'll need to:
 ---
 ### Step 3:  Configure the sample to use your Azure AD tenant
 
-In the steps below, "ClientID" is the same as "Application ID" or "AppId".
+In the steps below, "client-id" is the same as "Application ID" or "AppId".
+
+Open application.yml in your project to configure:
+
+```yml
+azure:
+  activedirectory:
+    tenantId: <your-tenantId>
+    client-id: <your-client-id>
+    client-secret: <your-client-secret>
+    # Optional, default value is http://localhost:8080/
+    redirectUriTemplate: <your-redirect-uri>
+    # groups that you created in your Azure AD tenant
+    user-group:
+      allowed-groups: group1,group2
+    # Optional, the default value is 
+    # environment: global  
+```
+
 
-Open application.properties in your project to configure:
 
-1. If your azure account follows format xxx@xxx.partner.onmschina.cn, configure property `azure.activedirectory.environment=cn` to use [Azure China](https://docs.microsoft.com/azure/china/china-welcome), the default value is `global`.
+1. If your azure account follows format xxx@xxx.partner.onmschina.cn, configure property `environment: cn` to use [Azure China](https://docs.microsoft.com/azure/china/china-welcome), the default value is `global`.
```

Comment:
Let's use the reference link style here, and put all our links at the bottom of this README file for better maintenance.